tools/releaseBuild/azureDevOps/templates/release-MakeContainerPublic.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+- download: none
+
+- template: release-SetReleaseTagAndContainerName.yml
+
+- pwsh: |
+    az login --service-principal -u $(az_url) -p $(az_key) --tenant $(az_name)
```

Comment:
We should declare what variable group we depend on for this job

Reply:
Added variables group in all jobs that use variables from groups

Reply:
fixed